packages/features/bookings/Booker/config.ts

```diff
@@ -124,6 +124,8 @@ export const useBookerResizeAnimation = (layout: BookerLayout, state: BookerStat
   useEffect(() => {
     const animationConfig = resizeAnimationConfig[layout][state] || resizeAnimationConfig[layout].default;
 
+    if (!animationScope.current) return;
```

Comment:
This should've been there in the first place, since refs can always be undefined. But now we don't always render the booker anymore, this definitely is the case and TS finally started to complain. So we check here if the ref is set, otherwise there's nothing to animate.